Earthfile

```diff
@@ -5,7 +5,7 @@ WORKDIR /usr/build
 
 build:
   COPY . .
-  RUN yarn && yarn run bootstrap
+  RUN yarn --immutable && yarn run bootstrap
```

Comment:
We also call it [here](https://github.com/replayio/replay-cli/blob/84e218d0cb05cbdbedda2e0d373ca332fbbbc9c7/.github/actions/playwright/action.yml#L22) in the example's dir. In there it probably doesn't matter but it might be best to pass it there explicitly too for consistency